packages/open-payments/src/client/index.ts

```diff
@@ -58,11 +58,14 @@ interface AuthenticatedRequestArgs {
    */
   accessToken: string
 }
-export interface ResourceRequestArgs
-  extends UnauthenticatedResourceRequestArgs,
+
+export interface GrantOrTokenRequestArgs
+  extends UnauthenticatedRequestArgs,
     AuthenticatedRequestArgs {}
 
-export interface CollectionRequestArgs extends AuthenticatedRequestArgs {
+export interface ResourceOrCollectionRequestArgs
```

Comment:
Just a thought I had, should we change the name of the `url` argument to be instead `resourceServerUrl` for the POST requests (since we end up appending the resource name, e.g. /incoming-payment), to be more clear that we _will_ be appending the resource name?

Reply:
Sorry, this 👆was supposed to be a comment to you question below.